app/pb_kits/playbook/pb_user_badge/user_badge.rb

```diff
@@ -7,9 +7,9 @@ class UserBadge
 
       partial "pb_user_badge/user_badge"
 
-      prop :badge, type: Playbook::Props::Enum,
-                   values: %w[million-dollar veteran],
-                   default: "million-dollar"
+      # prop :badge, type: Playbook::Props::Enum,
+      #              values: %w[million-dollar veteran],
+      #              default: "million-dollar"
```

Comment:
Kept this here bc we could still end up using this except without a default and more as a string instead potentially.